modules/monitoring-example-remote-write-authentication-settings.adoc

```diff
@@ -3,31 +3,41 @@
 // * observability/monitoring/configuring-the-monitoring-stack.adoc
 
 :_mod-docs-content-type: REFERENCE
-[id="example-remote-write-authentication-settings_{context}"]
-= Example remote write authentication settings
 
-// Set attributes to distinguish between cluster monitoring examples and user workload monitoring examples.
-ifndef::openshift-dedicated,openshift-rosa[]
+// The ultimate solution DOES NOT NEED separate IDs, it is just needed for now so that the tests will not break
+// tag::CPM[]
+[id="example-remote-write-authentication-settings-cpm_{context}"]
+= Example remote write authentication settings (CPM)
+// end::CPM[]
+
+// tag::UWM[]
+[id="example-remote-write-authentication-settings-uwm_{context}"]
+= Example remote write authentication settings (UWM)
```

Comment:
same as https://github.com/openshift/openshift-docs/pull/84002/files#r1823715863 for UWM

Reply:
https://84002--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/configuring-the-monitoring-stack#example-remote-write-authentication-settings-uwm_configuring-the-monitoring-stack
title is `Example remote write authentication settings (UWM)`